doc/sphinx/proof-engine/ltac.rst

```diff
@@ -572,7 +572,7 @@ Failing
    .. tacv:: gfail
       :name: gfail
 
-      This variant fails even if there are no goals left.
+      This variant fails when used after :n:`;` even if there are no goals left (see the example).
```

Comment:
I would rather say "This variant fails even when used after `n` and there are no goals left"